src/main/flight/pid.c

```diff
@@ -821,16 +837,29 @@ void FAST_CODE pidController(const pidProfile_t *pidProfile, const rollAndPitchT
 
             pidData[axis].D = pidCoefficient[axis].Kd * delta * tpaFactor;
 
+            float pidSetpointDelta = currentPidSetpoint - previousPidSetpoint[axis];
+
+#ifdef USE_RC_SMOOTHING_FILTER
+            if (axis == rcSmoothingDebugAxis) {
+                DEBUG_SET(DEBUG_RC_SMOOTHING, 1, lrintf(pidSetpointDelta * 100.0f));
```

Comment:
All these `DEBUG_SET` in `FAST_CODE` makes me think we should probably make `debug` `FAST_RAM`...